### PR TITLE
restrict size of letters

### LINKED ISSUE #2: creating very long memos (>16MB) will crash the server
this was brought up in this discussion: https://github.com/minetest/minetest/issues/13365. 

i may create a PR but i've got to evaluate how i want to do that. this mod is ancient and uses deprecated API.

### PR DESCRIPTION
**Pull request by fluxionary**
i opted to use the same limits as the books from minetest_game. 

fixes #2

### CODE REVIEW

File: locale/fr.txt
Comment (luk3yx) [inline locale/fr.txt:1]: Perhaps these should be converted to .tr files?

File: init.lua [hunk 1]
```diff
@@ -186,6 +181,8 @@ minetest.register_node("memorandum:letter_written", {
 		local item = sender:get_wielded_item()
 		if item:get_name() == "memorandum:eraser" then
 			local meta = minetest.get_meta(pos)
+			fields.text = fields.text:sub(1, 10000)
+			fields.signed = fields.signed:sub(1, 80)
```
Comment (luk3yx) [inline init.lua]:
```suggestion
			fields.text = (fields.text or ""):sub(1, 10000)
			fields.signed = (fields.signed or ""):sub(1, 80)
```

Comment (fluxionary) [reply init.lua]: there's a nil check for those values a couple lines up, is this still needed?

Comment (luk3yx) [reply init.lua]: I think that check is in a different `on_receive_fields` function

**Comment by luk3yx** [comment]
Thanks

**Pull request merged**